public/app/themes/justice/src/patch/js/ccfw-gtm.js

```diff
@@ -57,167 +69,189 @@ const CCFW = {
+            window.dataLayer = []
+        }
 
-        // INIT
-        // use existing list if present
-        // default to empty array if not
-        let allowedList = CCFW.storage.allowed.get() || []; // default to empty array
+      // INIT
+        let allowedList = CCFW.storage.allowed.get() || [] // default to empty array
 
-        //Always allow variables and triggers - https://developers.google.com/tag-manager/web/restrict
-        let ccfwTriggers = ['evl', 'cl', 'fsl', 'hl', 'jel', 'lcl', 'sdl', 'tl', 'ytl'];
-        let ccfwVariables = ['k', 'v', 'c', 'ctv', 'e', 'jsm', 'dpg', 'd', 'vis', 'gas', 'f', 'j', 'smm', 'r', 'remm', 'u'];
+      // Always allow variables and triggers - https://developers.google.com/tag-manager/web/restrict
+        let ccfwTriggers = ['evl', 'cl', 'fsl', 'hl', 'jel', 'lcl', 'sdl', 'tl', 'ytl']
+        let ccfwVariables = ['k', 'v', 'c', 'ctv', 'e', 'jsm', 'dpg', 'd', 'vis', 'gas', 'f', 'j', 'smm', 'r', 'remm', 'u']
 
-        allowedList = allowedList.concat(ccfwTriggers, ccfwVariables);
+        allowedList = allowedList.concat(ccfwTriggers, ccfwVariables)
 
-        window.dataLayer = [{
+        window.dataLayer.push({
             'gtm.allowlist': allowedList
-        }];
+        });
 
       // Drop GTM code
         (function (w, d, s, l, i) {
-            w[l] = w[l] || [];
+            w[l] = w[l] || []
             w[l].push({
                 'gtm.start':
                 new Date().getTime(), event: 'gtm.js'
-            });
+            })
             let f = d.getElementsByTagName(s)[0],
-            j = d.createElement(s), dl = l !== 'dataLayer' ? '&l=' + l : '';
-            j.async = true;
-            j.src = 'https://www.googletagmanager.com/gtm.js?id=' + encodeURIComponent(i) + dl;
-            f.parentNode.insertBefore(j, f);
-        })(window, document, 'script', 'dataLayer', CCFW.gtmID);
+            j = d.createElement(s), dl = l !== 'dataLayer' ? '&l=' + l : ''
+            j.async = true
+            j.src = 'https://www.googletagmanager.com/gtm.js?id=' + encodeURIComponent(i) + dl
+            f.parentNode.insertBefore(j, f)
+        })(window, document, 'script', 'dataLayer', CCFW.gtmID)
 
-        return true;
+        return true
     } else {
-        console.warn('CCFW GTM:', 'The GTM ID wasn\'t assigned or, does not exist.');
-        return false;
+        console.warn('CCFW GTM:', 'The GTM ID wasn\'t assigned or, does not exist.')
+        return false
+    }
+}
+
+const Toggle = function (e) {
+
+    if (typeof jQuery === undefined) {
+        console.log('jQuery is not defined')
+        return false
     }
-};
 
-const togglesChange = function (e) {
-    e.preventDefault();
-    let toggle = jQuery(this);
-    let allowed = toggle.data('allowlist');
-    let allowList = CCFW.storage.allowed.get() || [];
-    let pressed = toggle.attr('aria-checked') === 'true';
-    toggle.attr('aria-checked', !pressed);
+    e.preventDefault()
+    let toggle = jQuery(this)
```

Comment:
If we stick with semi-colons at line endings, it will be easier for upstream to review and merge.

Reply:
Thanks for pointing this out @EarthlingDavey 
My IDE strips them and then sometimes doesn't. It's weird behaviour.
When laravel mix does its work I think it adds them all anyway.
I'll try and fix it locally. Cheers

Reply:
Hey @EarthlingDavey I checked the [JavaScript standard and it states no semicolons](https://standardjs.com/rules#semicolons). Off the back of that, I've removed all semicolons in our patch code.

I'm more than happy to talk about this and hear your point of view if you'd like to.